CollapseLauncher/XAMLs/MainApp/Pages/GenshinGameSettingsPage.xaml.cs

```diff
@@ -12,14 +12,26 @@
 using RegistryUtils;
 using System;
 using System.IO;
+using System.Numerics;
+using System.Threading.Tasks;
+using Windows.Foundation;
 using static Hi3Helper.Locale;
 using static Hi3Helper.Logger;
 using static Hi3Helper.Shared.Region.LauncherConfig;
 using static CollapseLauncher.Statics.GamePropertyVault;
 using Hi3Helper;
 using Microsoft.UI.Xaml.Controls.Primitives;
 using Windows.Globalization.NumberFormatting;
+using Windows.Graphics.DirectX;
 using CollapseLauncher.Statics;
+using Microsoft.Graphics.Canvas.UI.Xaml;
+using Microsoft.Graphics.Canvas;
+using Microsoft.UI;
+using Windows.Storage.Streams;
+using Windows.Storage;
```

Comment:
Really minor, but ideally, we'd like to group the imports together, such that imports are grouped by packages.